modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/hash/PageMemoryHashIndexStorage.java

```diff
@@ -97,100 +93,88 @@ public HashIndexDescriptor indexDescriptor() {
 
     @Override
     public Cursor<RowId> get(BinaryTuple key) throws StorageException {
-        if (!closeBusyLock.enterBusy()) {
-            throwStorageClosedException();
-        }
+        return busy(() -> {
```

Comment:
Why did you have to do this? These are changes for the sake of changes. Did you want to artificially inflate you PR? Well, you succeeded in that case. Please roll it back or at least create a new "get*" method that would reduce the amount of unnecessary changes.
I feel like half of all changes in this PR are unjustified. I don't like it. Reviewing PRs like this one is a nightmare

Reply:
Tried to fix it.